src/popup.html

```diff
@@ -97,9 +150,13 @@ <h1 class="onboarding-h1"></h1>
                   <span class="emails-forwarded-text i18n-content" data-i18n-message-id="popupEmailsForwarded"></span>
                   <div class="dashboard-stats emails-forwarded"></div>
                 </li>
+                <li class="dashboard-info">
+                  <span class="email-trackers-removed-text i18n-content" data-i18n-message-id="popupEmailTrackersRemoved"></span>
+                  <div class="dashboard-stats email-trackers-removed"></div>
+                </li>
               </ul>
             </div>
-            <!-- Premium Dashboard-->
+            <!-- Register Domain Component -->
```

Comment:
Good work also cleaning up outdated comments. (Have I gone on one of my rants about default syntax highlighting themes always de-emphasising comments yet? :sweat_smile:)

Reply:
Thanks :) Also, could you slack me the behaviour you're talking about? Github's been having issues displaying videos.
<img width="509" alt="image" src="https://user-images.githubusercontent.com/13066134/187948847-4d2556f3-0b96-4f07-9618-22bec4c71844.png">